Mathlib/CategoryTheory/Functor/KanExtension/Basic.lean

```diff
@@ -388,6 +388,50 @@ end
 
 section
 
+variable (L : C ⥤ D) (F : C ⥤ H) (G : H ⥤ D')
+
+/-- Given a left extension `E` of `F : C ⥤ H` along `L : C ⥤ D` and a functor `G : H ⥤ D'`,
+`E.whiskerRight G` is the extension of `F ⋙ G` along `L` obtained by whiskering by `G`
+on the right. -/
+@[simps!]
+def LeftExtension.whiskerRight : LeftExtension L F ⥤ LeftExtension L (F ⋙ G) :=
```

Comment:
Should this be called `LeftExtension.postcomp₂` because it is analogous to `LeftExtension.postcomp₁`?